frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.tsx

```diff
@@ -76,7 +91,15 @@ export const DashCardMenu = ({
     },
   });
 
+  if (isDashCardMenuEmpty(plugins)) {
+    return null;
+  }
+
   const getMenuContent = () => {
+    if (typeof plugins?.dashboard?.dashcardMenu === "function") {
+      return plugins.dashboard.dashcardMenu({ question: question.card() });
+    }
```

Comment:
Do we expect SDK users to pass in a function or a React component in `dashcardMenu`? I was thinking how we could render `<DashcardMenu question={question} />` as well, if the dashcardMenu in `plugins.dashboard` is defined as a component.

(I saw you have `isValidElement` below to support rendering JSX, so I assume this is expected)